.eslintrc.json

```diff
@@ -0,0 +1,36 @@
+{
+    "env": {
+        "browser": true
+    },
+    "extends": [
+        "plugin:@typescript-eslint/recommended",
+        "plugin:react/recommended",
+        "plugin:prettier/recommended",
+        "prettier/react",
+        "prettier/@typescript-eslint"
+    ],
+    "settings": {
+      "react": {
+        "version": "detect"
+      }
+    },
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "project": "tsconfig.json",
+        "sourceType": "module"
+    },
+    "plugins": [
+        "@typescript-eslint",
+        "prettier",
+        "react"
+    ],
+    "rules": {
+      "@typescript-eslint/explicit-module-boundary-types": "off",
+      "@typescript-eslint/no-inferrable-types": "off",
+      "@typescript-eslint/no-explicit-any": "off",
+      "@typescript-eslint/no-non-null-assertion": "off",
+      "@typescript-eslint/ban-ts-comment": "off",
+      "react/prop-types": "off",
+      "@typescript-eslint/no-unused-vars": "off"
```

Comment:
Yes, it should probably be on, however I think I'll do that in a separate PR, since it was on before, so there are a lot of places that needs to be fixed first (which would pollute this PR I believe)

Reply:
Ok then. Can you create an issue regarding `@typescript-eslint/no-unused-vars` after merging this PR?

Reply:
I'll do

Reply:
https://github.com/cognitedata/gearbox.js/issues/651